doc/sphinx/source/quickstart/installation.rst

```diff
@@ -214,3 +232,15 @@ confirm that no errors are reported:
 .. code-block:: bash
 
     python setup.py test
+
+.. note::
+   Using ``pip`` as installation tool is very useful since it installs in place
+   (ie inside the current source code directory, and it does not perform a copy to
+   the target install directory ``site-packages``, unlike ``python setup.py install``, which
+   acts in an equivalent manner to ``make``); it also checks for needed dependencies
+   and, if need be, installs them on the fly, including Python wheels which are not installed via
+   ``python setup.py install``.
+
+.. note::
+    ``python setup.py test`` is the command to run most of the tests and should be performed
```

Comment:
This is already explained at the bottom of the document, a few lines below here, so I would not duplicate the information here.

Reply:
good call, removed it